celelstine/best-flight
# create endpoint to create user account

### Description
Delivers https://www.pivotaltracker.com/story/show/166494175, creates an endpoint to create user account.

### How to test
- make a post request to http://127.0.0.1:8000/api/v1/user/, ensure to enter valid data
```
{
        'user.email': [email],
        'user.first_name': [name[0]],
        'user.last_name': [name[1]],
        'user.password': [name[1]],
}
```
- should receive a response with about the your request.
